style.css

```diff
@@ -6,12 +6,12 @@
 
 body {
   margin: 0px;
-  color: ;
 }
 
 header {
   padding: 20px;
   background-color: #475b63;
+  display: grid;
```

Comment:
This is a good start, but you are missing the grid itself. Try adding some columns or rows ;)